test/repl.jl

```diff
@@ -358,37 +337,6 @@ temp_pkg_dir() do project_path; cd(project_path) do
         @test "Example" in c
         c, r = test_complete("rm Exam")
         @test isempty(c)
-        Pkg.REPLMode.pkgstr("develop $(joinpath(@__DIR__, "test_packages", "RequireDependency"))")
```

Comment:
Will this be brought back? This test is still useful and not `REQUIRE` specific

Reply:
It just needs a test package with a project file like this:
```
name = "Simple"
uuid = "632ccb1b-42db-4992-98c9-5cd245e24532"
version = "0.1.0"

[deps]
Example = "7876af07-990d-54b4-ab0e-23690620f79a"
```

Reply:
Added back.